requirements.txt

```diff
@@ -210,3 +210,4 @@ anyjson==0.3.3
 pylama==6.4.0
 # sha256: yY7zECUyGAxtDrno1pd0u97Y2QtnGYjuRV1W_h9xGos
 django-guardian==1.3
+-e git://github.com/jotes/django-pg-fts.git@django_1.8.4#egg=django-pg-fts
```

Comment:
I've tried following thing:
```
git+https://github.com/jotes/django-pg-fts.git@django_1.8.4#egg=django-pg-fts==0.1.1
```
But peep returned an error:
```
django-pg-fts==0.1.1 from git+https://github.com/jotes/django-pg-fts.git@django_1.8.4 (from -r requirements.txt (line 215)): The download link, git+https://github.com/jotes/django-pg-fts.git@django_1.8.4, would not result in a file that can be hashed. Peep supports only == requirements, file:// URLs pointing to files (not folders), and http:// and https:// URLs pointing to tarballs, zips, etc.
```
I can upload tarball somewhere but i have doubts if it's the best way. Do you have any preferences or suggestions in that regard?